content/docs/explanation/models/data-models-full-example/index.fr.md

```diff
@@ -67,94 +67,135 @@ Pour toutes les sections de voies de notre infrastructure, les attributs `geo` e
 
-##### Retour aux aiguillages
+![Diagramme des positions de l'aiguillage de croisement double](svg_diagrams/DoubleSlipCrossing_A2toB1.svg) ![Diagramme des positions de l'aiguillage de croisement double](svg_diagrams/DoubleSlipCrossing_A2toB2.svg)
 
-Un `Switch` possède trois attributs :
+**4) L'aiguillage de croisement simple**
 
-* `switch_type` : l'identifiant [`SwitchType`](#types-daiguillages) de cette aiguillage.
+Celui-ci ressemble plus à un mélange entre une aiguille simple et un croisement. Il possède quatre ports : *A1*, *A2*, *B1* et *B2*.
+
+![Diagramme d'un aiguillage de croisement simple](svg_diagrams/DoubleSlipCrossing_A1toB1.svg)
+
+Voici les trois connexions que peut réaliser cet aiguillage :
+
+- *A1* vers *B1*
+- *A1* vers *B2*
+- *A2* vers *B2*
+
+![Diagramme des positions de l'aiguillage de croisement simple](svg_diagrams/SingleSlipCrossing_A1toB1.svg) ![Diagramme des positions de l'aiguillage de croisement simple](svg_diagrams/SingleSlipCrossing_A1toB2.svg)
+![Diagramme des positions de l'aiguillage de croisement simple](svg_diagrams/SingleSlipCrossing_A2toB2.svg)
+
+**5) Le lien entre deux sectionx de voies**
+
+Celui-ci représente le lien entre deux sections de voies. Il possède deux ports : *A* et *B*.
+
+![Diagramme d'un lien](svg_diagrams/link.svg)
+
+ Il permet de créer un lien entre deux sections de voies tel que définis dans OSRD. Ce `NodeType` n'est pas un objet physique.  
+
+##### Retour aux noeuds
+
+Un `Node` possède trois attributs :
+
+* `node_type` : l'identifiant [`NodeType`](#types-daiguillages) de ce noeud.
 * `ports` : une correspondance entre les noms de port et les extrémités des sections de voie.
 * `group_change_delay` : le temps qu'il faut pour changer le groupe de l'aiguillage qui est actif.
 
-Les noms des ports doivent correspondre aux ports du type d'aiguillage choisi. Les extrémités de la section de voie peuvent être début ou fin, faites attention à choisir les bonnes.
+Les noms des ports doivent correspondre aux ports du type du noeud choisi. Les extrémités de la section de voie peuvent être début ou fin, faites attention à choisir les bonnes.
 
-La plupart des aiguillages de notre exemple sont des aiguillages habituels. Le chemin de la gare du Nord à la gare du Sud a deux aiguillages de croisement, et il y a un aiguillage de croisement double juste avant que la ligne principale ne se divise en lignes Nord-Est et Sud-Est.
+La plupart des noeuds de notre exemple sont des noeuds habituels. Le chemin de la gare du Nord à la gare du Sud a deux aiguillages de croisement, et il y a un aiguillage de croisement double juste avant que la ligne principale ne se divise en lignes Nord-Est et Sud-Est.
 
 ![Diagramme des sections de voie et des aiguillages](svg_diagrams/small_infra_rails_n_points.drawio.en.svg)
 
+Il est important de noter que ces noeuds sont présents en dur dans le code du projet. Seul les `extended_switch_type` ajoutés par l'utilisateur apparaîtront dans le railjson. 
```

Comment:
I'll turn *en dur* by *par défaut* 
This part is not present in the English translation.